docbook5-book/xml/generic-entities.ent

```diff
@@ -103,24 +103,24 @@
 <!ENTITY slea                   "SLE">
 <!ENTITY sleda                  "SLED">
 <!ENTITY slsa                   "SLES">
-<!ENTITY slehaa                 "SLE-HA">
-<!ENTITY slehaga                "SLE-HA-GEO"> <!-- This one is not officially defined. -->
-<!ENTITY slehpca                "SLE-HPC">
-<!ENTITY sles4sapa              "SLES-SAP">
+<!ENTITY slehaa                 "SLE HA">
+<!ENTITY slehaga                "SLE HA GEO"> <!-- This one is not officially defined. -->
+<!ENTITY slehpca                "SLE HPC">
```

Comment:
I'd suggest to use a non-breakable space instead of a space:

```suggestion
<!ENTITY slehaa                 "SLE&#xa0;HA">
<!ENTITY slehaga                "SLE&#xa0;HA&#xa0;GEO"> <!-- This one is not officially defined. -->
<!ENTITY slehpca                "SLE&#xa0;HPC">
```